src/crypto.rs

```diff
@@ -30,11 +30,14 @@ fn generichash_quick(msg: &[u8], key: Option<&[u8]>) -> Result<[u8; 32]> {
         .expect("generichash returned result of wrong size"))
 }
 
-pub fn init() -> Result<()> {
+pub(crate) fn init() -> Result<()> {
```

Comment:
Instead of putting `pub(crate)` everywhere here, should we maybe just import the module as `pub(crate)`?

Reply:
The module is already crate-local, putting `pub(crate)` here only has the effect of making it obvious that this is the case. If these types *were* publicly exposed somewhere, this wouldn't compile.

Reply:
OK, good then. :)

It's just that `pub(crate)` is a pet-peeve of mine. It's too annoying to type/read for something that should be used often.

Reply:
I find it quite useful to immediately see if some item is public API or not. If it's declared `pub(crate)`, it cannot possibly be reexported publicly.

Reply:
Yeah, I agree. I just wished it was less ugly than `pub(crate)`.

Reply:
That's fair. Did you know you can make it even uglier by putting an arbitrary (parent) module path into those parentheses? https://doc.rust-lang.org/reference/visibility-and-privacy.html#pubin-path-pubcrate-pubsuper-and-pubself